### DISCUSSION TITLE
connections/v7/admin/secure/c_azure_oidc_container

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#27] [Q&A]
# connections/v7/admin/secure/c_azure_oidc_container

Single sign-on is accomplished by setting up a trust relationship between the Connections server and Microsoft Azure Active Directory using the IBM WebSphere OpenID Connect Relying Party Trust Association Interceptor (OIDC Relying Party TAI).

https://help.hcltechsw.com/connections/v7/admin/secure/c_azure_oidc_container.html

**Comment by barbj** [comment]
Azure Active Directory is now Microsoft Entra ID :
https://www.microsoft.com/en-us/security/business/identity-access/microsoft-entra-id

Here is an example of what google cloud did: https://cloud.google.com/architecture/identity/federating-gcp-with-azure-ad-configuring-provisioning-and-single-sign-on
